content/advanced-use.md

```diff
@@ -0,0 +1,104 @@
+# Advanced use
+
+This section of the CLI Guides describes how to make custom configurations to the app's build behavior, testing, and more.
+Ember is designed for a zero config experience for most users, but it was also designed to be extensible. Think of it as configuration without confusion.
+
+## Overall CLI functionality
+
+To begin making configurations, first it is important to understand the overall architecture of what the CLI can do.
+Most custom configuration is done in `ember-cli-build.js`, `environment.js`, and Broccoli plugins.
+
+### Broccoli
+
+Ember uses Broccoli for the build process. Broccoli is an independent project that is similar to tools like webpack and parcel. Common Broccoli configurations include adding assets into the build pipeline. For example, if an app has content in the form of Markdown files that need to be turned into HTML during the build, a Broccoli plugin would be the way to go. This very app you're reading content on right now follows that architecture.
+
+Just like there are Ember community addons, there are a variety of Ember community broccoli plugins too!
+
+### Babel
+
+Babel is an independent project used basically everywhere on the web. Browsers all have different behavior for handling the connections between JavaScript modules. Babel converts the modules into something that the wide variety of browsers can understand. The CLI uses Babel as part of the build process.
```

Comment:
My knowledge of Babel is limited, but I don’t think it’s primarily about JavaScript module formats—and that’s the sense I get reading this paragraph. It’s more about JavaScript language features in general. Maybe changing module formats is one of the types of transformation it can do?